server/components/filters/utils.test.ts

```diff
@@ -0,0 +1,198 @@
+import Utils from './utils'
+import { FieldDefinition } from '../../types/reports'
+import Dict = NodeJS.Dict
+import { FilterType } from './enum'
+import { DateRange } from './types'
+
+const selectFieldFormat: Array<FieldDefinition> = [
+  {
+    header: 'Select Field',
+    name: 'selectField',
+    filter: {
+      type: FilterType.select,
+      options: [
```

Comment:
Minor and optional: The options array could be extracted to a variable since it is the same used twice in the selectFieldFormat and the radioFieldFormat.